9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init ]; then
```

Comment:
I have tried to write something like this. It was about how to get script into image. So I try to broaden it a bit.